### PR TITLE
add support for Axon40 and G_IDB60

### PR DESCRIPTION
**Pull request by npspears**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/checkerboards/g_idb60/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# G-IDB60 
+
+60% drop-in PCB for IDB60 feauturing Tsangan bottom row, split right shift, and split backspace. 
+
+- Keyboard Maintainer: [Nasp](https://github.com/npspears)
+- Hardware Supported: G-IDB60 PCB
+- Hardware Availability: https://www.checkerboards.xyz/gidb60.html
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/checkerboards/g_idb60/readme.md:9]: Could you add instructions on how to enter the bootloader here? 


Eg, is there a hardware reset button, etc.

Comment (npspears) [reply keyboards/checkerboards/g_idb60/readme.md:9]: Done

**Review by npspears — COMMENTED** [review]
added bootloader reset instructions

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]
In future, please raise separate PRs for each keyboard.

**Pull request merged**